Project.toml

```diff
@@ -33,23 +33,23 @@ DocStringExtensions = "0.5, 0.6, 0.7, 0.8, 0.9"
 FFMPEG = "0.2, 0.3, 0.4"
 GeometryBasics = "0.3, 0.4"
 HTTP = "1"
+Meshing = "0.5, 0.6"
 MeshIO = "0.4"
-Meshing = "0.5"
 MsgPack = "1"
-Parameters = "0.10, 0.11, 0.12"
+Parameters = "0.10, 0.11, 0.12, 0.13"
 PrecompileTools = "1"
-Requires = "0.5, 1"
-Rotations = "1.3"
-StaticArrays = "0.10, 0.11, 0.12, 1"
+Requires = "1"
+Rotations = "1"
+StaticArrays = "1"
```

Comment:
I deleted all the 0.x versions for `StaticArrays`. Is there any advantage in keeping them?